src/zampy/datasets/land_cover.py

```diff
@@ -205,34 +205,36 @@ def extract_netcdf_to_zampy(file: Path) -> xr.Dataset:
     Returns:
         Coarse land cover data in zampy format.
     """
-    with TemporaryDirectory() as temp_dir:
+    with TemporaryDirectory(ignore_cleanup_errors=True) as temp_dir:
```

Comment:
I don't think this is a good idea. It could lead to files not being cleaned up (on windows), and filling up the user's temp directory with unused zampy files...

Reply:
Do you run the tests locally on WSL or Windows? You should be able to debug this locally instead of trying to do it through the CI :smile:

Reply:
> Do you run the tests locally on WSL or Windows? You should be able to debug this locally instead of trying to do it through the CI 😄

I tested it on WSL and everything works nicely. I agree that maybe it is nice to configure my windows environment and test it locally. 👍

Reply:
> I don't think this is a good idea. It could lead to files not being cleaned up (on windows), and filling up the user's temp directory with unused zampy files...

Thanks for the heads up. It was a suggestion from [here](https://www.scivision.dev/python-tempfile-permission-error-windows/). I agree that we need to be careful about it. I will remove it then.